src/Workspaces/CSharp/Portable/Simplification/Reducers/CSharpMiscellaneousReducer.cs

```diff
@@ -110,14 +111,61 @@ public CSharpMiscellaneousReducer() : base(s_pool)
             OptionSet optionSet,
             CancellationToken cancellationToken)
         {
-            if (node.Statements.Count == 1 &&
-                CanHaveEmbeddedStatement(node.Parent) &&
-                !optionSet.GetOption(CSharpCodeStyleOptions.PreferBraces).Value)
+            if (node.Statements.Count != 1)
             {
-                return node.Statements[0];
+                return node;
             }
 
-            return node;
+            if (!CanHaveEmbeddedStatement(node.Parent))
+            {
+                return node;
+            }
+
+            switch (optionSet.GetOption(CSharpCodeStyleOptions.PreferBraces).Value)
+            {
+                case PreferBracesPreference.Always:
+                    return node;
```

Comment:
i would prefer having a default-clause that falls into this case.

Reply:
➡️ Now added